static/app/components/modals/debugFileCustomRepository/appStoreConnect/index.tsx

```diff
@@ -46,6 +46,7 @@ type ItunesRevalidationSessionContext = SessionContext & {
 };
 
 type InitialData = {
```

Comment:
i'm actually left wondering if it's possible to just export `CustomRepoAppStoreConnect` so instead duplicating its fields here, we can just use `Omit<CustomRepoAppStoreConnect, "details">`. 

it's not immediately obvious that `InitialData` is just `CustomRepoAppStoreConnect`, so it can be a little troublesome to figure out which bits of the backend provide this data.